stix_shifter_modules/trustar/stix_transmission/api_client.py

```diff
@@ -0,0 +1,376 @@
+from stix_shifter_utils.stix_transmission.utils.RestApiClient import RestApiClient
+from stix_shifter_utils.utils.error_response import ErrorResponder
+from .error_mapper import ErrorMapper
+import requests
+import requests.auth
```

Comment:
not sure why do you import requests library here?

Reply:
@mdazam1942
Implemented as suggested review it and merge it.